content/docs/command-reference/exp/clean.md

```diff
@@ -0,0 +1,33 @@
+# exp clean
+
+Cleanup internal DVC experiment temporary files.
+
+## Synopsis
+
+```usage
+usage: dvc exp clean [-h] [-q | -v]
+```
+
+## Description
+
+Runs housekeeping tasks within the DVC repository, such as removing outdated
```

Comment:
minor: I think `DVC repository` can be wrapped into `<abbr>` (I would check something similar in docs)